static/app/components/onboardingWizard/onboardingCard.tsx

```diff
@@ -0,0 +1,137 @@
+import React, {Fragment} from 'react';
+import styled from '@emotion/styled';
+import {motion} from 'framer-motion';
+import {PlatformIcon} from 'platformicons';
+
+import Card from 'sentry/components/card';
+import Link from 'sentry/components/links/link';
+import {IconChevron, IconEllipsis} from 'sentry/icons';
+import {t, tct} from 'sentry/locale';
+import pulsingIndicatorStyles from 'sentry/styles/pulsingIndicator';
+import space from 'sentry/styles/space';
+import {Organization, Project} from 'sentry/types';
+import trackAdvancedAnalyticsEvent from 'sentry/utils/analytics/trackAdvancedAnalyticsEvent';
+import withProjects from 'sentry/utils/withProjects';
+import {usePersistedOnboardingState} from 'sentry/views/onboarding/targetedOnboarding/types';
+
+const MAX_PROJECT_COUNT = 3;
+
+function OnboardingViewTask({
+  org,
+  projects: allProjects,
+}: {
+  org: Organization;
+  projects: Project[];
+}) {
+  if (!org?.experiments.TargetedOnboardingMultiSelectExperiment) {
+    return null;
+  }
+
+  const [onboardingState] = usePersistedOnboardingState();
+  if (!onboardingState) {
+    return null;
+  }
+  // Projects selected during onboarding but not received first event
+  const projects = onboardingState.selectedPlatforms
+    .map(platform => onboardingState.platformToProjectIdMap[platform])
+    .map(projectId => allProjects.find(p => p.slug === projectId))
+    .filter(project => project && !project.firstEvent) as Project[];
+  return projects.length > 0 ? (
+    <Fragment>
+      <Heading>{t('Setup Projects')}</Heading>
+      <TaskCard key="onboarding-continue-card">
+        <OnboardingTaskProjectList>
+          {projects.slice(0, MAX_PROJECT_COUNT).map(p => (
+            <OnboardingTaskProjectListItem
+              key={p.id}
+              to={`/onboarding/${org.slug}/setup-docs/?project_id=${p.id}`}
+              onClick={() => {
+                trackAdvancedAnalyticsEvent('growth.onboarding_quick_start_cta', {
+                  platform: p.platform,
+                  organization: org,
+                });
+              }}
+            >
+              <OnboardingTaskProjectListItemInner>
+                <PlatformIcon platform={p.platform || 'default'} />
+                {p.slug}
+                <PulsingIndicator />
+                <PulsingIndicatorText>{t('Waiting for event')}</PulsingIndicatorText>
+                <IconChevron direction="right" />
+              </OnboardingTaskProjectListItemInner>
+            </OnboardingTaskProjectListItem>
+          ))}
+          {projects.length > MAX_PROJECT_COUNT && (
+            <OnboardingTaskProjectListItem
+              to={`/onboarding/${org.slug}/setup-docs/`}
+              onClick={() => {
+                trackAdvancedAnalyticsEvent('growth.onboarding_quick_start_cta', {
+                  organization: org,
+                });
+              }}
+            >
+              <OnboardingTaskProjectListItemInner>
+                <IconEllipsis />
```

Comment:
Feels like a weird usage of IconEllipsis, since usually this is used for an overflow menu. cc @doralchan

Reply:
Also @jmanhart